### PR TITLE
create release command and release channels rpc call

### PR DESCRIPTION
**Pull request by martineehrlich**
## Description
A user should be able to close channels on a market.
- Create CLI command
- Call closeChannels on base and counter engines

## Related PRs
https://github.com/sparkswap/lnd-engine/pull/89

## Todos
- [x] Tests
- [x] Documentation
- [x] Link to Trello

### CODE REVIEW

**Comment by martineehrlich** [comment]
[wallet release command for broker](https://trello.com/c/2DQWfnO9/474-wallet-release-command-for-broker)

File: broker-daemon/broker-rpc/wallet-service/release-channels.js [hunk 1]
```diff
@@ -0,0 +1,43 @@
+const { PublicError } = require('grpc-methods')
+
+/**
+ * Grabs public lightning network information from relayer and opens a channel
+ *
+ * @param {Object} request - request object
+ * @param {Object} request.params
+ * @param {RelayerClient} request.relayer
+ * @param {Logger} request.logger
+ * @param {Engine} request.engines
```
Comment (dannypaz) [inline broker-daemon/broker-rpc/wallet-service/release-channels.js]: missing orderbooks here

File: broker-daemon/broker-rpc/wallet-service/release-channels.js [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+ * @param {Object} responses
+ * @param {function} responses.EmptyResponse
```
Comment (dannypaz) [inline broker-daemon/broker-rpc/wallet-service/release-channels.js]: change to `ReleaseChannelsResponse`

File: broker-daemon/broker-rpc/wallet-service/release-channels.js [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+ * @return {responses.EmptyResponse}
+ */
+async function releaseChannels ({ params, relayer, logger, engines, orderbooks }, { ReleaseChannelsResponse }) {
+  const { market } = params
+  const orderbook = orderbooks.get(market)
+
+  if (!orderbook) {
+    throw new PublicError(`${market} is not being tracked as a market.`)
+  }
+  const [ baseSymbol, counterSymbol ] = market.split('/')
```
Comment (dannypaz) [inline broker-daemon/broker-rpc/wallet-service/release-channels.js]: @martineehrlich is `engines.get` case sensitive? We may have to upcase `baseSymbol` and `counterSymbol` as param checking?

Comment (dannypaz) [reply broker-daemon/broker-rpc/wallet-service/release-channels.js]: @martineehrlich we decided that if the user provides the incorrect currency symbol that these errors are sufficient for now

**Pull request merged**